tools/webpack-jumpstart

```diff
@@ -17,6 +17,18 @@ set -eu
 cd "$(realpath -m "$0"/../..)"
 . tools/git-utils.sh
 
+wait=''
+while [ $# != 0 ] ; do
```

Comment:
Let's see when we move to `getopt(1)` :grin:

Reply:
Honestly, it doesn't do a whole lot...